templates/_language_picker_modal.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n %}
+
+{% get_language_info_list for LANGUAGES as languages %}
+<div id="languagePickerModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">{% trans "Select language" context "Change language modal title" %}</h5>
+      </div>
+      <form id="language-picker" action="{% url 'set_language' %}" method="POST">
+      <div class="modal-body">
+          {% csrf_token %}
+          <input name="next" type="hidden" value="{{ request.get_full_path }}">
+          {% for language in languages %}
+            <div class="radio">
+              <label>
+                <input type="radio" name="language" value="{{ language.code}}" {% if language.code == LANGUAGE_CODE %}checked{% endif %}>
+                {{ language.name_local }}
```

Comment:
Some languages are starting with a capital, some don't (e.g.: French is "français" because in French language name in sentences don't start with a capital; Then we have "English" which does start with a capital in sentences).

We should add a CSS rule or add the filter `capfirst`.

Reply:
I'm not sure this is a problem. In fact it could be considered a grammar error to spell "polski" with a capital letter unless at the start of a sentence.

Reply:
That's interesting. Alright then!